packages/cli/src/index.ts

```diff
@@ -117,6 +125,24 @@ for (const appType of ['block', 'theme']) {
         });
 }
 
+cli.command('verify-manifest', 'verify the manifest structure to be correct')
```

Comment:
Do we really need this command? as it is ran also on build.

Reply:
ah yeeh, i was just thinking if someone wants to verifing before hes deploying, to be sure. But probably something that never someone uses xD